pandas/io/formats/latex.py

```diff
@@ -657,7 +694,27 @@ def _select_builder(self) -> Type[TableBuilderAbstract]:
         return TabularBuilder
 
     @property
-    def column_format(self) -> str:
+    def caption(self) -> str:
+        return self._caption
+
+    @caption.setter
+    def caption(self, caption: Optional[Union[str, Tuple[str, str]]]) -> None:
+        if caption:
+            if isinstance(caption, str):
+                self._caption = caption
+                self.short_caption = ""
+            else:
+                try:
+                    self._caption, self.short_caption = caption
+                except ValueError as err:
+                    msg = "caption must be either str or tuple of two strings"
+                    raise ValueError(msg) from err
```

Comment:
Can you add a test that hits this?

Reply:
It is there, if you mean raising ValueError.

String 561.
```
        # test that wrong number of params is raised
        with pytest.raises(ValueError):
            df.to_latex(caption=(the_caption, the_short_caption, "extra_string"))
```

Reply:
I updated the test to ensure that the error message matches expectations.